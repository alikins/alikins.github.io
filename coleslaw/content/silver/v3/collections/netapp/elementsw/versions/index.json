{
    "meta": {
        "count": 1
    },
    "links": {
        "first": "example/content/silver/v3/collections/netapp/elementsw/versions/index.json",
        "last": "example/content/silver/v3/collections/netapp/elementsw/versions/index.json",
        "next": null,
        "prev": null
    },
    "data": [
        {
            "version": "20.2.0",
            "href": "example/content/silver/v3/collections/netapp/elementsw/versions/20.2.0/",
            "created_at": "2020-06-06T21:47:56.520375",
            "modified_at": "2020-06-06T21:47:56.520375"
        }
    ]
}