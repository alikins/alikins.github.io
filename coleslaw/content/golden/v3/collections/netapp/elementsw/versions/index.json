{
    "meta": {
        "count": 1
    },
    "links": {
        "first": "example/content/golden/v3/collections/netapp/elementsw/versions/index.json",
        "last": "example/content/golden/v3/collections/netapp/elementsw/versions/index.json",
        "next": null,
        "prev": null
    },
    "data": [
        {
            "version": "20.2.0",
            "href": "example/content/golden/v3/collections/netapp/elementsw/versions/20.2.0/",
            "created_at": "2020-06-06T21:47:56.520375",
            "modified_at": "2020-06-06T21:47:56.520375"
        }
    ]
}